### ISSUE TITLE
Pyenv virtualenv packages sync between users

### ISSUE THREAD

**Issue by Alexgar24** [macdub/pyenv-multiuser#9]
Hi! First and foremost, thank you for the plugin; it's truly commendable.

If I understand correctly, when a user activates a virtual environment and uses pip to install a package, that package is only accessible to the user who installed it. I tested this by creating an empty environment and installing numpy with one user account using `pip install numpy`. Subsequently, I checked from another account using `pip freeze` and noticed that numpy wasn't listed.

Is this behavior intentional? If not, addressing this might enhance the plugin significantly.

Thanks!

**Comment by macdub** [comment]
Hi @Alexgar24

This plugin shouldn't have any impact on the modules that are installed into a virtual environment. By default, pyenv has a 'shims' directory in the PYENV_ROOT path. The `multiuser` plugin allows for each user to setup a shims directory outside of the PYENV_ROOT and the user owns that directory. So there won't be any write permission issues as well as shims being overwritten should another user come and source a different environment with different packages installed.

At any rate, I will take a look and see if there is something going on. Could you provide the explicit steps that you took to produce this issue?

Would you confirm that you sourced the same virtual environment from the secondary account that you ran `pip freeze` from?

**Comment by Alexgar24** [comment]
Hi!

I'm not sure I explained correctly before. The reason why I installed this plugin was to avoid shims permissions errors when many people using and installing packages on the same pyenv virtualenvs. What I expected of the plugin was that if I installed a new package in venv, it will be accessible for everyone using that same venv. What I've found is that that if I install a new package it is only accessible to me, since the shim for it is in my home directory. And I wonder if the package should be actually doing what I expected or what it currently does.

And I do confirm I was comparing the same venvs.

Thank you!
